### PR TITLE
Ability to display and cleanup old DB records

### LINKED ISSUE #121: Ability to delete reports older than specified time interval
As an admin I need to be able to delete reports that are older than specified time interval.

Acceptance criteria:
- new CLI flag to specify "delete old reports" operation
- old reports are first print into logs
- ability to just print the reports without the real `DELETE` operation

See https://github.com/RedHatInsights/insights-results-aggregator-cleaner/

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Ability to display and cleanup old DB records

Fixes #121

## Type of change

- New feature (non-breaking change which adds functionality)
- Documentation update

## Testing steps

N/A

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: differ/storage.go [hunk 1]
```diff
@@ -557,9 +598,9 @@ func (storage DBStorage) CleanupForOrganization(orgID types.OrgID, maxAge string
 
 	log.Info().
 		Str(MaxAgeAttribute, maxAge).
-		Str("delete statement", printableStatement).
+		Str("DeleteStatement", printableStatement).
```
Comment (epapbak) [inline differ/storage.go:601]: DeleteStatement without the double quota

Comment (tisnik) [reply differ/storage.go:601]: ah sure

File: differ/storage.go [hunk 2]
```diff
@@ -584,7 +625,7 @@ func (storage DBStorage) CleanupForOrganization(orgID types.OrgID, maxAge string
 //
 // The method return number of deleted records.
 func (storage DBStorage) CleanupNewReportsForOrganization(orgID types.OrgID, maxAge string) (int, error) {
```
Comment (epapbak) [inline differ/storage.go:627]: We don't have any way to call this one, right?

File: differ/storage.go [hunk 3]
```diff
@@ -597,7 +638,7 @@ func (storage DBStorage) CleanupNewReportsForOrganization(orgID types.OrgID, max
 //
 // The method return number of deleted records.
 func (storage DBStorage) CleanupOldReportsForOrganization(orgID types.OrgID, maxAge string) (int, error) {
```
Comment (epapbak) [inline differ/storage.go:640]: We don't have any way to call this one, right?

File: differ/storage.go (hunk 2 above)
Comment (tisnik) [reply differ/storage.go:627]: nope AFAIK, it was prepared for weekly report functionality, where this cleanup was planned

File: differ/storage.go (hunk 3 above)
Comment (tisnik) [reply differ/storage.go:640]: nope AFAIK, it was prepared for weekly report functionality, where this cleanup was planned

File: differ/storage.go (hunk 2 above)
Comment (epapbak) [reply differ/storage.go:627]: Let's keep them until we have some input on that feature then =)

**Review by epapbak — APPROVED** [review]

**Pull request merged**